### DISCUSSION TITLE
How do you style the main 'body' or 'root' id?

### DISCUSSION THREAD

**Discussion by raspharaoh** [marimo-team/marimo#793] [Q&A] [answered]
Looking at the source html, after the app is rendered in the webpage there is:
 ```html
   <body>
    <div id="root"></div>
   </body>
```
How can I access the main page to style it?

**Comment by mscolnick** [comment, marked as answer]
you could try:

```python
mo.Html("""
    <style>
      body {
         ...
      }
    </style>
""")
```

but as a heads up, we don't guarantee any component classnames as "public API", meaning they could change and break css selectors
